.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Substrate Connect CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-extension-versions:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - id: package-version
+      run: echo "::set-output name=version::`grep -rw './projects/extension/package.json' -e 'version' | sed 's/[^0-9.]//g'`"
+    - id: webpack-version
+      run: echo "::set-output name=version::`grep -rnw './projects/extension/webpack.config.js' -e 'PKG_VERSION' | grep -o '".*"' | tr -d '"'`"
+    - id: manifest-version
+      run: echo "::set-output name=version::`grep -rw './projects/extension/public/manifest.json' -e 'version' | sed 's/[^0-9.]//g'`"
+    - run: exit 1
+      if: ${{ steps.package-version.outputs.version != steps.webpack-version.outputs.version }}
```

Comment:
Maybe add a third check where we make sure that `version` is not `""`, so that we detect if the CI breaks.

Reply:
I do not think this is needed - as for this to happen one needs to remove versions in all 3 files and I assume that will be noticed in the PR